### PR TITLE
Fix issue generations detecting running cron jobs in the last 24 hrs.

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

File: .github/actions/testing_report_generation/app.rb [hunk 1]
```diff
@@ -116,11 +116,13 @@ def get_workflows(client, repo_name)
     else
       failure_report.add_workflow_run_and_result(workflow_text, result_text)
     end
+  else
+    failure_report.add_workflow_run_and_result(workflow_text, "The cron job did not run inthe last 24 hrs.")
```
Comment (paulb777) [inline .github/actions/testing_report_generation/app.rb:120]: s/inthe/in the/

File: .github/actions/testing_report_generation/app.rb [hunk 2]
```diff
@@ -116,11 +116,13 @@ def get_workflows(client, repo_name)
...
   end
 end
 
 # Check if there exists any cron jobs.
-if failure_report.get_report.nil? && success_report.get_report.nil?
+if failure_report.get_report.nil? && success_report.get_report.nil
```
Comment (paulb777) [inline .github/actions/testing_report_generation/app.rb:125]: Is this intended?

Comment (granluo) [reply .github/actions/testing_report_generation/app.rb]: Fixed

File: .github/actions/testing_report_generation/app.rb (hunk 1 above)
Comment (granluo) [reply .github/actions/testing_report_generation/app.rb]: Fixed

**Comment by granluo** [comment]
Looks like the pull request does not trigger the job with the updated yml. Could this PR be merged and test with another PR?

**Review by paulb777 — APPROVED** [review]
Sure.

**Pull request merged**